images/utils/influxdb_data_migration.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if [ -d ${OLD_DATA_DIR}/influxdb ]; then 
```

Comment:
Still need this?

Reply:
Yeah, I am checking what should be deleted. Should at least one more commit with unneeded files removal. Stuck on testing because quay.io is unstable and cannot allow me to build containers.